release/models/aft/openconfig-aft-common.yang

```diff
@@ -348,40 +517,45 @@ submodule openconfig-aft-common {
       default false;
       description
         "Flag that controls pop action, i.e., the top-most MPLS label
-         should be popped from the packet when switched by the system.
+        should be popped from the packet when switched by the system.
 
         The top-most MPLS label associated with pop action is equal to
-        the label key used in 'mpls' AFT 'label-entry' list.";
+        the label key used in 'mpls' AFT 'label-entry' list.
+
+        This node must be supported in addition to the
```

Comment:
I don't think this is true based on your comments above. I think this node will not be deprecated.

Reply:
Yes, we'll leave decap to a future PR and if that results in this leaf being deprecated, note it then.  So I removed this text.